shift-shift.go

```diff
@@ -242,9 +249,17 @@ func listenKeyboards(
 				case 1: // key down
 					switch ev.Code {
 					case keyFirst:
-						groupFirst = true
+						if dblKeystroke {
+							t, groupFirst = pressedBefore(t, dblKeyTimeout)
```

Comment:
this function has too many variables already so it'd be cool to have a meaningful name of the variable instead of one-letter variable `t`, maybe `doublePressTimer` ?

Reply:
I'd keep it, bit it's up to @grafov